src/Sylius/Bundle/ApiBundle/Exception/NotFoundExceptionInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Exception;
+
+interface NotFoundExceptionInterface extends \Throwable
```

Comment:
Extending the top-level exception interface is a really bad idea.

Reply:
@stloyd could you explain why ? I was inspired by this interface : `Symfony\Component\Serializer\Exception\ExceptionInterface`